IntegrationTests/Package.swift

```diff
@@ -83,9 +70,25 @@ func addClientRuntimeDependency() {
 }
 
 func addAWSClientRuntimeDependency() {
-    package.dependencies += [
-        .package(path: "../../aws-sdk-swift")
-    ]
+    let smithySwiftURL = "https://github.com/awslabs/aws-sdk-swift"
```

Comment:
This local var is called `smithySwiftURL` but it contains the aws-sdk-swift URL

Reply:
Also, can we combine this logic with the smithy-swift logic? (i.e. one case statement where two package dependencies are added in each case)

Reply:
Oof nice catch, and sure, one minute